applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -756,7 +779,7 @@ static void on_cloud_evt_cell_pos_data_received(const struct nrf_cloud_data *con
 			LOG_INF("TTFF %ds", (int)k_uptime_delta(&ttft_start)/1000);
 			ttft_start = 0;
 		}
-		rsp_send("\r\n#XCELLPOS: %d,%lf,%lf,%d\r\n",
+		rsp_send("\r\n#XLOCATION: %d,%lf,%lf,%d\r\n",
```

Comment:
Should cell based location have the old unsolicited notification XCELLPOS and only wifi based location uses XLOCATION? In this way we can keep backward compatibility of old AT-functionality.

Reply:
That is possible, then it would become
`#XCELLPOS: ....`
`#XWIFIPOS: ....`
respectively

Reply:
Yes, I would prefer this way :)